clang-tools-extra/clangd/XRefs.cpp

```diff
@@ -2238,7 +2238,10 @@ prepareCallHierarchy(ParsedAST &AST, Position Pos, PathRef TUPath) {
   for (const NamedDecl *Decl : getDeclAtPosition(AST, *Loc, {})) {
     if (!(isa<DeclContext>(Decl) &&
           cast<DeclContext>(Decl)->isFunctionOrMethod()) &&
-        Decl->getKind() != Decl::Kind::FunctionTemplate)
+        !(Decl->getKind() == Decl::Kind::Var &&
+          !cast<VarDecl>(Decl)->isLocalVarDecl()) &&
+        Decl->getKind() != Decl::Kind::FunctionTemplate &&
```

Comment:
nit: can can you keep the function template check next to the function check, since they are related?

Reply:
Allright, I initially put them apart because I thought this arrangement was easier to read, but I understand why it makes sense to keep it there.